src/main/java/com/cmsc495/ticketsystem/model/MyUser.java

```diff
@@ -0,0 +1,59 @@
+/*MyUser class represents a user and is used define and create myUser objects */
+
+package com.cmsc495.ticketsystem.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class MyUser {
```

Comment:
This looks good to me but lets call it UserModel to stick to our established design convention